safe_relay_service/relay/services/transaction_service.py

```diff
@@ -398,7 +398,9 @@ def estimate_circles_signup_tx(
         """
         # Tx data from Circles Hub contract `signup` method
         data = "0xb7bc0f73"
-        return 1800000000000000
+        estimated = self.estimate_circles_hub_method(data, safe_address, gas_token)
+        logger.info("estimated value for token deployment: %s", estimated)
+        return 1900000000000000
```

Comment:
so it's calculated bu the estimates function and then it returns  fixed value?

Reply:
yes because I tried only with the estimated value and it doesn't work, so I am debugging still...